roles/ceph-osd/tasks/osd_fragment.yml

```diff
@@ -50,7 +50,7 @@
   with_items: combined_osd_id.results
 
 - name: copy ceph.conf for assembling
```

Comment:
I think ansible can render variables in `name:` so it would be nice to see this as:

    - name: copy {{ cluster }}.conf for assemling